eng/pipelines/vmr-internal-sync.yml

```
@@ -0,0 +1,33 @@
+pr: none
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - internal/release/*
+
+resources:
+  repositories:
+  - repository: vmr
+    type: git
+    name: dotnet-dotnet
+    ref: $(Build.SourceBranch)
+
+parameters:
+- name: vmrBranch
+  displayName: dotnet-dotnet branch to push to
+  type: string
+  default: ''
+
+variables: 
+- ${{ if ne(parameters.vmrBranch, '') }}:
```

Can we make this change so that the parameter has an actual default?

```suggestion
  default: ' '

variables: 
- ${{ if ne(parameters.vmrBranch, ' ') }}:
```